bci_tester/data.py

```diff
@@ -1029,6 +1029,13 @@ def create_BCI(
     custom_entry_point="/bin/bash",
 )
 
+PYTORCH_CONTAINER = create_BCI(
+    build_tag=f"{SAC_CONTAINER_PREFIX}/pytorch:2.5.0",
```

Comment:
You will have to bump this string on every version bump in pytorch, which seems extremely wasteful to me. Can you use a more generic tag? Or are you really expecting to stay at 2.5.0 for an extended period of time?